src/views/tiles/NodeView.js

```diff
@@ -98,15 +100,14 @@ exports = Class(ImageView, function (supr) {
 			}
 
 			var hideViews = {};
-			for (var tag in itemViews) {
-				var itemView = itemViews[tag];
+			for (tag in itemViews) {
+				itemView = itemViews[tag];
 				if (itemView.style.visible) {
 					hideViews[tag] = itemView;
 				}
-			};
+			}
 
-			var itemView;
-			for (var tag in tile.tags) {
```

Comment:
all the above are jshint fixes,

removing duplicate declarations and unnecessary semicolon